packages/editor-ui/src/components/RunData.vue

```diff
@@ -845,7 +846,7 @@ export default defineComponent({
 
 			return inputData;
 		},
-		inputData(): INodeExecutionData[] {
+		inputDataAll(): INodeExecutionData[] {
```

Comment:
Nit: The naming strikes me as a bit confusing, I'd find this clearer:
- `inputDataAll()` → `inputData()` (live or pin data)
- `inputData()` → `inputDataPage()` (only a slice)